### PR TITLE
feat: Implement dotnet6

### PR DESCRIPTION
**Pull request by lamonfly**
GitHub Issue: #

## Proposed Changes

Feature

## What is the new behavior?

Add .net 6

## Checklist

Please check if your PR fulfills the following requirements:

- [X] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [ ] Contains **NO** breaking changes
- [ ] Updated the Changelog
- [ ] Associated with an issue

### CODE REVIEW

File: Src/Shared/BiometryService.WinUI.cs [hunk 1]
```diff
@@ -0,0 +1,240 @@
+﻿#if WINDOWS_UWP
```
Comment (jeanplevesque) [inline Src/Shared/BiometryService.WinUI.cs:1]: The file is called xxx.WinUI.cs but the `#if` doesn't reflect that.

File: Src/BiometryService.Uno.WinUI/BiometryService.Uno.WinUI.csproj [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <PackageReference Include="System.Reactive" Version="5.0.0" />
+    <PackageReference Include="Uno.Core" Version="4.0.1" />
+    <PackageReference Include="Uno.WinUI" Version="4.4.13" />
+    <PackageReference Include="Uno.SourceGenerationTasks" Version="4.1.0-dev.9" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)'=='net6.0-android'">
+    <PackageReference Include="Xamarin.AndroidX.Biometric" Version="1.1.0.10" />
+    <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.5.1" />
+    <PackageReference Include="Xamarin.AndroidX.Core" Version="1.9.0" />
+    <PackageReference Include="Xamarin.AndroidX.RecyclerView " Version="1.2.1.7" />
```
Comment (carlh98) [inline Src/BiometryService.Uno.WinUI/BiometryService.Uno.WinUI.csproj:30]: remove space

**Review by carlh98 — APPROVED** [review]

File: src/BiometryService.SampleApp.Uno/BiometryService.SampleApp.Uno.Shared/MainPage.xaml.cs [hunk 3]
```diff
@@ -13,7 +19,7 @@
 using BiometryService.SampleApp.Uno.Droid;
 using AndroidX.Biometric;
 #endif
-#if WINDOWS_UWP
+#if WINDOWS_UWP || WINDOWS
```
[the remark this answers is not in the document]
Comment (lamonfly) [reply src/BiometryService.SampleApp.Uno/BiometryService.SampleApp.Uno.Shared/MainPage.xaml.cs:22]: This is a shared MainPage across all Samples, including UWP and Windows WinUI. In this part both UWP and Windows are used.

File: src/BiometryService.SampleApp.Uno/BiometryService.SampleApp.Uno.Shared/MainPage.xaml.cs [hunk 4]
```diff
@@ -62,7 +72,11 @@ public MainPage()
             else
             {
 			    _biometryService = new BiometryService(MainActivity.Instance,
-												   Dispatcher,
+#if WINUI
```
[the remark this answers is not in the document]
Comment (lamonfly) [reply src/BiometryService.SampleApp.Uno/BiometryService.SampleApp.Uno.Shared/MainPage.xaml.cs:75]: Same here.

**Review by Iguins — APPROVED** [review]

**Pull request merged**